2024-03-27/minutes.md

```diff
@@ -1 +1,431 @@
+
+**Host:** Brian (@bsmth)
+
+## Intro & overview of agenda
+
+(Brian)
+Welcome, great to see everyone including new faces.
+Agenda:
+
+- new Footer that's on the website.
+- community participation guidelines (CPG)
+- What's changed on Discord and Matrix
+- The new blog post out
+
+I will try and leave as much time as possible for discussions and questions because we have few at the end.
+
+## Team updates
+
+- The Matrix room and the general channel on Discord are bridged.
+  That's a bi-directional chat. So messages post in the MDN matrix room, posted in the general channel on discord and vice versa.
+  It's 2-way communication to make sure that people on Matrix are not completely left out and that we don't miss messages there because we tend to be in Discord quite a lot - it's tricky being everywhere. This should help it a little; we're going to see how how that works out.
+- And we have a new blog post, it's about Interop: https://developer.mozilla.org/en-US/blog/interop2023-mdn-doc-updates/
+
+### Footer
+
+[Ruth John]
+We've updated the MDN article footer based on user feedback, shifting more from decisions based on personal preferences to data-driven choices.
+This involves a simple feedback form (on-page) asking users if they found the content helpful, so we're working more towards improving content quality through user insights.
+Although there's no freeform text field feedback (for qualitative data), we're exploring other methods to gather this.
+
+Since launching, we've seen an average 70% positive sentiment in March.
+We will to analyze this data by larger sections before tackling individual pages, combining it with other metrics like page views to inform updates or removals.
+This is an ongoing project, and we're not taking specific requests yet for the data as we focus on actionable tasks. Any questions?
+
+[Estelle] Q: Is there any way for people outside of Mozilla to see the feedback that is coming through?
+
+[Ruth John]
+No, the requests will come in, I know. That's why I would ask you not to ask me for that just yet, I want to be able to analyze the data first and be able to answer follow-up questions, which we'll surely have.
+
+[Estelle]
+Side note: I really miss the edit button in the Footer and I realized that I can just click on the "view on GitHub" and then edit.
+
+[Ruth John]
+Yeah, we went back and forth on this a lot of times. There were 3 links all to exactly the same thing in the old footer and it just it didn't make any sense.
+
+### Community Participation Guidelines
+
+[Brian Smith]
+I'll talk about Mozilla's Community Participation Guidelines (CPG) — this is super important for us, especially because we're on GitHub all day and working closely with the community.
+Here's a link for anyone interested.
```

Comment:
need to add the link here

Reply:
added in 461fb2eb90c49dbcdbd4d01ac922895e56d0c976